opengl_ros_lib/src/depth_image_projector.cpp

```diff
@@ -68,6 +68,10 @@ struct DepthImageProjector::Impl
     //not const parameters
     std::array<float, 2> depthFocalLength_ = {};
     std::array<float, 2> depthCenter_ = {}; 
+    std::array<float, 16> depthToMap_ = {0.0f, -1.0f,  0.0f, 0.0f,
+                                         0.0f,  0.0f, -1.0f, 0.0f,
+                                         1.0f,  0.0f,  0.0f, 0.0f,
+                                         0.0f,  0.0f,  0.0f, 1.0f};
```

Comment:
This array initialization in struct causes error when compiled with gcc 5.3, which is the newest version distributed with Ubuntu 16.04. Although this is [a bug of gcc](https://stackoverrun.com/ja/q/9068088), could you add another brace for work around to be successfully compiled with Ubuntu 16.04 environment?